cmds/less/less.go

```diff
@@ -72,7 +72,7 @@ type Lesser struct {
 
 	// searchResults are the results for the current search.
 	// They should be highlighted.
-	searchResults searchResults
+	searchResults *searchResults
```

Comment:
Why this change?

Reply:
`go vet` complains when a mutex is copied by value. `searchResults` contains a mutex in there somewhere which `go vet` is smart enough to complain about as well. The fix is to copy the pointer rather than the value, but `go vet` still seems to be complaining. I'm still looking into this...

Reply:
NewSearchResults also returns a copy. It should probably return a pointer anyways given the name,

Reply:
Thank you! I changed a bunch of values to pointers all over the code, and it seems to work. Do you know why it was an issue to copy a mutex by value?

Reply:
It is not safe to copy a sync.Mutex once it is in use. It contains internal fields about waiters that lose meaning once copied (since the waiters are waiting on the old value). Plus it just doesn't make much sense. If you are copying a Mutex, how do other users know about the new location.

In this case, the copies were all before the first use, but the linter doesn't know that and just warns regardless.

https://godoc.org/sync#Mutex